tests/conftest.py

```diff
@@ -12,3 +12,17 @@ def redis_client() -> redis.Redis:
 @pytest.fixture()
 def redis_message_broker(redis_client: redis.Redis) -> RedisMessageBroker:
     return RedisMessageBroker(client=redis_client, channel_prefix="test_diator_channel")
+
+@pytest.fixture()
+async def rabbitmq_client() -> aio_pika.Connection:
```

Comment:
It would be better to use:
https://testcontainers-python.readthedocs.io/en/latest/rabbitmq/README.html - for RabbitMQ
https://testcontainers-python.readthedocs.io/en/latest/redis/README.html - for Redis

As a result, there will be no need for the docker-compose.